src/containers/linksDisplay/linksDisplay.jsx

```diff
@@ -0,0 +1,114 @@
+// @flow
+import React, { Component } from "react";
+import styled from "styled-components";
+import { connect } from "react-redux";
+import { links as linksActions } from "../../actions/index";
+import TableLink from "./tableLink";
+import TableList from "./tableList";
+import Alert from "../../components/alert";
+import ControlPanel, {
+  NavLink,
+  ResetIndicator,
+  ResetButton,
+  TableTypeButton,
+  ListTypeButton
+} from "./controlPanel";
+
+const DisplayWrapper = styled.div`
+  width: 100%;
+`;
+type Link = {
+  url: string,
+  shortLink: string,
+  passage: number,
+  tags: Array<string>,
+  description: string
+};
+
+type Props = {
+  error: string,
+  linksList: ?Array<Link>,
+  loading: boolean,
+  loadAllLinks: () => void,
+  loadMyLinks: () => void,
+  typeLoad: "my" | "all"
+};
+type State = {
+  typeDisplay: "table" | "list"
```

Comment:
Don't repeat yourself in codding, move types into const at the top of file, and re-use it from there